### ISSUE TITLE
Allow for fish of known origin in the mixture

### ISSUE THREAD

**Issue by eriqande** [eriqande/rubias#12]
Terry Beacham noted that they are going to be doing joint PBT/GSI on many of their samples, and
often they have parentage assignments that are nearly exact for fish that are in the mixture.  

It seems to me a good way to approach this is to:

1. Do PBT and identify they fish in each mixture that you can by parentage.
2. Feed all the fish in the GSI analysis, but flag the ones from known origin.
3. Then, we compute the Q-matrix for all fish.  However, after we have done that,
if there are any fish of known origin from PBT, we modify their rows (or columns?) in the
Q matrix to be zeroes (or 1e-50's) everywhere except for their correct row which 
gets a 1.  

Then we do GSI as before.  That individual will show up with posterior of 1 of coming from its correct population, and it is also there to inform the estimate of pi.

**Comment by eriqande** [comment]
I am actively working on this now.  I have started work in a local branch called `known-indivs-in-mixtures` which I have branched off of `add-haploid-markers`.  

I think the input file should have a column called `known_collection` which must be identical to `collection` for the reference sample fish, but for the mixture fish should be NA or should be one of the collections that exists in the reference (will make a function `check_known_collections` to enforce things).
